migrations/versions/b49314aac7b0_create_stints_for_missing_users.py

```diff
@@ -0,0 +1,103 @@
+"""create stints for missing people
+
+Revision ID: b49314aac7b0
+Revises: 105986a70a5d
+Create Date: 2022-02-07 23:55:08.293544
+
+"""
+from alembic import op
+from sqlalchemy.sql import table, column
+from sqlalchemy import String, Integer, Date
+from datetime import timedelta
+
+from backend.models import Stint
+
+
+# revision identifiers, used by Alembic.
+revision = "b49314aac7b0"
+down_revision = "105986a70a5d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    stint_table = table(
+        "stint",
+        column("stint_id", Integer),
+        column("profile_id", Integer),
+        column("type_stint", String),
+        column("start_date", Date),
+        column("end_date", Date),
+        column("title", String),
+    )
+
+    connection = op.get_bind()
+
+    """
+    The 156 nicetys with null dates can be grouped into 18 profiles
```

Comment:
I think this is 156 niceties with null *stint_id*, rather than null dates, right?